CSharp/ArticleProjects/ApplicationInsights/Microsfot.Azure.Batch.Samples.TelemetryStartTask/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.ApplicationInsights.Extensibility;
```

Comment:
Do the other samples/article projects have usings inside or outside the namespace? From what I can see it looks like we've got it inside for most samples? Update yours to be inside as well?